lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -36,6 +38,9 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
         for chapter in response.data['courseware_summary']:
             self.assertIsNotNone(chapter)
         self.assertIn('settings/grading/' + str(self.course.id), response.data['studio_url'])
+        self.assertEqual(response.data['credit_support_url'], CREDIT_SUPPORT_URL)
+        self.assertIsNotNone(response.data['verification_data'])
+        self.assertNotEqual(response.data['verification_data']['status'], '')
```

Comment:
Does anything change about these for an Audit vs Verified learner? You can add a 
```
if enrollment_mode == CourseMode.AUDIT:
    self.assert...
elif enrollment_mode == CourseMode.VERIFIED:
    self.assert...

Reply:
Anything about credit requirements? Is there an eligibility message you could verify on?